Content.Server/GameObjects/Components/Photography/PhotoCameraComponent.cs

```diff
@@ -0,0 +1,333 @@
+            }
+
+            CameraOn = true;
+            _spriteComponent.LayerSetState(1, "camera_on");
+            _audioSystem.PlayFromEntity("/Audio/machines/machine_switch.ogg", Owner, AudioHelpers.WithVariation(0.15f).WithVolume(-5));
+            return true;
+        }
+
+        public bool UseEntity(UseEntityEventArgs eventArgs)
+        {
+            return ToggleOnOff();
+        }
+
+        public void Examine(FormattedMessage message, bool inDetailsRange)
+        {
+            if (CameraOn)
+            {
+                message.AddMarkup(Loc.GetString("The {0} is [color=green]On[/color]", Owner.Name));
+            }
+            else
+            {
+                message.AddMarkup(Loc.GetString("The {0} is [color=red]Off[/color]", Owner.Name));
+            }
+
+            if (inDetailsRange)
+            {
+                message.AddMarkup(Loc.GetString("\nFilm: [color={0}]{1}/{2}[/color], ",
+                    !HasFilm ? "red" : "white", Film, FilmMax));
+                message.AddMarkup(Loc.GetString("Radius: [color=white]{0}x{0}[/color]", Radius*2));
+            }
+        }
+
+        public bool InteractUsing(InteractUsingEventArgs eventArgs)
+        {
+            if (!eventArgs.Using.TryGetComponent<PhotoFilmComponent>(out var film))
+                return false;
+
+            var empty = film.TakeFilm(FilmMax - Film, out var took);
+            Film += took;
+            if (empty)
+            {
+                film.Owner.Delete();
+            }
+            if (took > 0)
+            { 
+                _audioSystem.PlayFromEntity("/Audio/machines/machine_switch.ogg", Owner, AudioHelpers.WithVariation(0.15f).WithVolume(-5));
+            }
+            return true;
+        }
+
+        public SuicideKind Suicide(IEntity victim, IChatManager chat)
+        {
+            if (HasFilm && CameraOn) 
+            {
+                //You could say it was a... Fatal Frame™
+                PlaySoundCollection("TakePhoto", -5);
+                chat.EntityMe(victim, Loc.GetString("takes {0:their} last photo! It looks like {0:theyre} trying to commit suicide!", victim));
+                SendNetworkMessage(new SuicideSelfieMessage(victim.Uid)); //will eventually delete the victim
+                return SuicideKind.Special;
+            }
+            chat.EntityMe(victim, Loc.GetString("beats {0:themselves} to death with the {1}!", victim, Owner.Name));
+            return SuicideKind.Brute;
+        }
+
+        [Verb]
+        public sealed class PhotoRadius2x2 : Verb<PhotoCameraComponent>
+        {
+            protected override void GetData(IEntity user, PhotoCameraComponent component, VerbData data)
+            {
+                data.CategoryData = VerbCategories.PhotoRadius;
+                data.Text = "2x2";
+            }
+
+            protected override void Activate(IEntity user, PhotoCameraComponent component)
+            {
+                component.Radius = 1;
```

Comment:
These verbs need to check whether the user is actually holding the camera and not incapacitated or such.

Reply:
D'oh.